lom/src/pluginmgr/pluginmgr_common/pluginmgr_helper.go

```diff
@@ -173,12 +203,14 @@ func (plmgr *PluginManager) run() error {
             for {
                 serverReq, err := plmgr.clientTx.RecvServerRequest()
                 if plmgr.getShutdownStatus() {
-                    lomcommon.LogInfo("In run() RecvServerRequest: Shutdown is active, ignoring request: %v", serverReq)
-                    //return
+                    LogInfo("In run() RecvServerRequest: Shutdown is active, ignoring request: %v", serverReq)
```

Comment:
Should we not return the function here even in production scenario as plugin manager is shutting down ?

Reply:
Any how plugin manager will exit. Its better to log a message to convey that engine is sending requests despite sending shutdown. So if exit is there here, then this logging will not be achieved.

Reply:
When the system is shutdown only, this code is executed. This is for the unit tests. Also here there is no performance hit as this gets called only when system is shutdown.

Reply:
if lomcommon.GetLoMRunMode() == lomcommon.LoMRunMode_Test {
                        return
                    }

remove this and return logerror

Reply:
TODO